ocaml/xapi/dbsync_slave.ml

```diff
@@ -408,5 +409,13 @@ let update_env __context sync_keys =
         Xapi_host.set_console_idle_timeout ~__context ~self:localhost
           ~value:console_timeout
   ) ;
-
+  switched_sync Xapi_globs.sync_secure_boot (fun () ->
+      let result =
+        try
+          let contents = Unixext.string_of_file !Xapi_globs.secure_boot_path in
+          contents.[4] <> '\x00'
+        with _ -> false
```

Comment:
I think we should log if this fails for any reason because we expect this not to fail.